src/js/edu-benefits/utils/validations.js

```diff
@@ -212,6 +212,17 @@ function isValidSeparatedDateField(date, dateEntered) {
   return true;
 }
 
+function isValidRepayingPeriodToDate(date, dateStarted) {
```

Comment:
I was mostly copying what had been done before, but that makes sense to me.

I was playing around with doing this in moment, but wasn't sure if there was some reason we avoided it for these validations in hca.